### DISCUSSION TITLE
The code gets aborted or shows bug error

### DISCUSSION THREAD

**Discussion by kumaranvenk** [neperfepx/neper#788] [Q&A] [answered]
I have updated the latest available version 4.7.0. I plotted for 12 seed points, can you suggest how to restrict the domain in above case, because, when I am trying to plot, blank images are coming.

The code I used: neper -T -n 164 -dim 2 -domain "square(800,800) -morphooptiin"coo:file(coo),weight:file(wts)" -statcell "area"

some data from coo1 file:
1369.45 424.162
477.995 562.569
822.255 468.65

some data from wts file:
6.7703
29.6186
14.7555
27.4547
8.8129

**Comment by rquey** [comment, marked as answer]
I would need the full data files.
